nav2_costmap_2d/include/nav2_costmap_2d/inflation_layer.hpp

```diff
@@ -77,7 +78,7 @@ class InflationLayer : public Layer
 public:
   InflationLayer();
 
-  ~InflationLayer() override = default;
+  ~InflationLayer();
```

Comment:
Why remove the override default? I think someone did that for linters

Reply:
Because I needed to override it to delete the mutex